### PR TITLE
Add Indonesian Beverage

### PR DESCRIPTION
**Pull request by KangAbbad**

### CODE REVIEW

File: src/recipes/wedang-uwuh.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "name": "Wedang Uwuh",
+    "description": "traditional drink originating from the Javanese tribe",
+    "github": "@KangAbbad",
```
Comment (alfg) [inline src/recipes/wedang-uwuh.json:4]:
```suggestion
    "github": "KangAbbad",
```

File: src/recipes/wedang-uwuh.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        {
+            "quantity": "10",
+            "measure": "oz",
+            "ingredient": "rock sugar"
+        },
+        {
+            "quantity": "2",
+            "measure": "cup",
+            "ingredient": "water"
+        },
```
Comment (alfg) [inline src/recipes/wedang-uwuh.json:30]:
```suggestion
        }
```

File: src/recipes/wedang-uwuh.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "Also put all the clove, cinnamon and rock sugar in it.",
+        "Boil for 15-20 minutes, wait for it.",
+        "And wedang uwuh ready to be served."
+    ],
+    "image": "wedang-uwuh.jpg",
+    "keywords": [
+        "wedang uwuh",
+        "indonesian beverage",
+        "traditional",
+        "javanese",
```
Comment (alfg) [inline src/recipes/wedang-uwuh.json:44]:
```suggestion
        "javanese"
```

**Review by alfg — CHANGES_REQUESTED** [review]
Added few suggestions. Thanks.

**Comment by KangAbbad** [comment]
> Added few suggestions. Thanks.

**Comment by alfg** [comment]
Closing as this conflicts with another recipe added recently.

**Pull request closed without merging**